blog/2020-04-07-ModernJavaConstraintsNoDRL.adoc

```diff
@@ -0,0 +1,153 @@
+= Modern java constraints (no Drools Rule Language) - ConstraintStreams
```

Comment:
How about `Constraint Streams: Modern Java constraints without the Drools Rule Language`.

To me, this reads better and in a more logical way.
Also, I believe that `Java` is capitalized and the entire documentation uses `Constraint Streams` as two words. I too prefer it that way.